tests/benchmark.py

```diff
@@ -448,6 +448,7 @@ def main():
         nargs="+",
         choices=known_libraries,
         help=("Remove specified libraries from the benchmarks"),
```

Comment:
Whilst we're here, we can remove the redundant parentheses here and in the next `help`:
```suggestion
        help="Remove specified libraries from the benchmarks",
```

And some typos a bit further down:

`disabled_libraires` -> `disabled_libraries`

And:

```diff
-Ensure the modules are avilable in a the global scope
+Ensure the modules are available in the global scope
```

Reply:
done

Reply:
Thanks, and could you remove the parentheses on the next `help` too?